### DISCUSSION TITLE
SmugglerDatabaseApi - Unable to read data from the transport connection

### DISCUSSION THREAD

**Discussion by roklenardic** [ravendb/ravendb#15861] [General]
When executing the SmugglerDatabaseApi.Between method in order to sync some collections (trying with a single collection with 2 items right now), I get the following error after about 10mins of Smuggler chasing down all collections:
System.IO.IOException: Unable to read data from the transport connection: The connection was closed.

Stacktrace screenshot:
![image](https://user-images.githubusercontent.com/121102/220370634-74f6acc8-47ca-479e-88d6-30dfd9efb223.png)

So even if I specify a single collection to a single List<FilterSetting> it still crunches all the collections, transfers nothing, and fails after about 10 mins.

Am I missing a setting so that Smuggler could focus on exactly collection I want to sync?

**Comment by garayx** [comment]
I guess the reason for this may be the size of the request or the generation time of the request. If you are using IIS you can check IIS logs or IIS config. Please note that this version is out of support.

**Comment by ayende** [comment]
This version is not supported, mind.
You may have better luck here if you'll do separate export & import steps
